content/en/docs/best-practices/general-principles/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: General Principles
```

Comment:
This seems more like "Multicluster Principles" than general to me?

Reply:
I would think a general principles one would be more along the lines of make sure you have proper HA setup (with info about how to do this), fine tuning resource requests/limits, etc

Reply:
Agreed, these are really deployment principles rather than general best practice principles.

I think a few folks have pointed this out.  @rcaballeromx should we rename this doc?

Reply:
@nmittler @howardjohn would this suggestion work better?
```suggestion
title: Deployment Best Practices
```